client/src/lib/components/Cards/CollectionCards/CollectionFullCard.tsx

```diff
@@ -66,13 +67,13 @@ const Row = styled.View({
 
 const LeftColumn = styled.View({
   flex: 1,
-  height: 86,
   paddingVertical: SPACINGS.EIGHT,
 });
 
 const TitleWrapper = styled.View({
   flexDirection: 'row',
   alignItems: 'center',
+  flex: 1,
```

Comment:
fixes this: 
<img width="367" alt="Screenshot 2023-09-12 at 10 13 32" src="https://github.com/29ki/29k/assets/7523828/6c97a616-3de9-4227-b8df-0793297616f7">